examples/webgl_camera.html

```diff
@@ -243,21 +226,30 @@
 
 				cameraRig.lookAt( mesh.position );
 
-				renderer.clear();
+				//
 
 				activeHelper.visible = false;
 
+				renderer.setClearColor( 0x000000, 1 );
```

Comment:
Actually I find the usage of different clear colors a bit confusing since it seems the example shows two different scenes (and not just the same scene rendered with different cameras).

Reply:
OK, but now I am inclined to think there is no reason to make any changes at all. We are just making it unnecessarily confusing...

Reply:
Right, looking at the code now users probably ask themselves why the same clear color is configured two times.

Reply:
When thinking about it, two different clear color values was indeed better. In this way the scissor configuration has an immediate purpose.

Reply:
I'll restore what I originally submitted.